easybuild/easyconfigs/t/TRIQS/TRIQS-2.2.0-foss-2019a-Python-2.7.15.eb

```diff
@@ -0,0 +1,55 @@
+easyblock = 'CMakeMake'
+
+name = 'TRIQS'
+version = '2.2.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'https://triqs.github.io/'
+description = """
+ TRIQS (Toolbox for Research on Interacting Quantum Systems) is a
+ scientific project providing a set of C++ and Python libraries to
+ develop new tools for the study of interacting quantum systems.
+"""
+
+docurls = ['https://triqs.github.io/triqs/%(version_major_minor)s.x/reference.html']
+software_license = 'LicenseGPLv3'
+
+toolchain = {'name': 'foss', 'version': '2019a'}
+toolchainopts = {'pic': True, 'usempi': True}
+
+sources = ['triqs-%(version)s.tar.gz']
+source_urls = ['https://github.com/TRIQS/triqs/releases/download/%(version)s/']
+checksums = ['5f7fca3dd140d86e464ab18ff91fa16febad865344ec2c5b924434f507ef4038']
+
+dependencies = [
+    ('Python', '2.7.15'),
+    ('Boost', '1.70.0'),
+    ('Clang', '8.0.0'),
+    ('GMP', '6.1.2'),
+    ('HDF5', '1.10.5'),
+    ('Mako', '1.0.8'),
+    ('h5py', '2.9.0')
+]
+
+builddependencies = [
+    ('CMake', '3.13.3')
+]
+
+separate_build_dir = True
+
+runtest = 'test'
+
+sanity_check_paths = {
+    'files': ['lib/libtriqs.%s' % SHLIB_EXT],
+    'dirs': ['bin', 'include', 'lib', 'share'],
```

Comment:
@rokzitko I think it's worth enhancing this a little bit, to check for some specific binaries, and also check for `lib/python%(pyshortver)s/site-packages` rather than just `lib`:

```python
sanity_check_paths = {
    'files': ['bin/triqs++', 'lib/libtriqs.%s' % SHLIB_EXT],
    'dirs': ['include/triqs', 'lib/python%(pyshortver)s/site-packages', 'share'],
}
```

Maybe it's also worth adding `triqs++ --help` as an additional sanity check command?

```python
sanity_check_commands = [
    "triqs++ --help",
    "python -c 'import pytriqs'",
]